services/frontend-service/src/ui/components/SideBar/SideBar.tsx

```diff
@@ -235,17 +237,19 @@ export const SideBar: React.FC<{ className?: string; toggleSidebar: () => void }
             });
             setLockMessage('');
         }
-        api.batchService()
-            .ProcessBatch({ actions })
-            .then((result) => {
-                deleteAllActions();
-                showSnackbarSuccess('Actions were applied successfully');
-            })
-            .catch(() => {
-                showSnackbarError('Actions were not applied. Please try again');
-            });
-        handleClose();
-    }, [actions, api, handleClose, lockCreationList, lockMessage]);
+        if (authReady) {
+            api.batchService()
+                .ProcessBatch({ actions }, authHeader)
+                .then((result) => {
+                    deleteAllActions();
+                    showSnackbarSuccess('Actions were applied successfully');
+                })
+                .catch(() => {
```

Comment:
🔧 can we log the error on console.error? otherwise it is quite hard to figure out what went wrong?

Reply:
good point, done